crates/aptos-logger/src/aptos_logger.rs

```diff
@@ -208,13 +208,13 @@ impl LogEntry {
         let peer_id: Option<&str>;
         let chain_id: Option<u8>;
 
-        #[cfg(node_identity)]
+        #[cfg(feature = "node-identity")]
         {
             peer_id = aptos_node_identity::peer_id_as_str();
             chain_id = aptos_node_identity::chain_id().map(|chain_id| chain_id.id());
         }
 
-        #[cfg(not(node_identity))]
+        #[cfg(not(feature = "node-identity"))]
```

Comment:
@rustielin i'm not sure if this feature is used?  But, I made sure it was done correctly

Reply:
fyi, @aluon